Python/Tests/Core/ProcessOutputTests.cs

```diff
@@ -37,39 +37,39 @@ public class ProcessOutputTests {
         [TestMethod, Priority(UnitTestPriority.P1)]
         public void ArgumentQuoting() {
             foreach (var testCase in new[] {
-                new { Source = "Abc", Expected = "Abc" },
+ new { Source = "Abc", Expected = "Abc" },
```

Comment:
Spacing seems off? Is it supposed to be with no tabs?

Reply:
updated